__tests__/app.test.js

```diff
@@ -294,3 +307,16 @@ describe("Patch /api/articles/1", () => {
       });
   });
 });
+describe("Delete comments", () => {
+  test("Delete a certain comment by it's id", () => {
+    return request(app).delete("/api/comments/1").expect(204);
+  });
+  test("should return 400 if comment id is valid but does not exist", () => {
```

Comment:
Your test description refers to 404 where the ID is valid, but the implementation is testing 400 with an invalid ID. I recommend to adjust this description to invalid ID check, and add another test for non-existent ID.